app/views/admin/menu/_menu_pages.html.erb

```diff
@@ -0,0 +1,13 @@
+  <%= render partial: 'admin/menu/menu__section_start', locals:
+      { text: 'Pages', icon: 'list-rich' } %>
+      <%= render partial: 'admin/menu/menu__item', locals:
```

Comment:
Consider introducing a helper method to make these repeated calls a little more succinct, e.g.

```diff
-<%= render partial: 'admin/menu/menu__item', locals: { text: 'List Pages', link: ..., icon: '...' } %>
+<%= render_menu_item 'List pages', link: ... icon: '...' %>
```